src/config.ts

```diff
@@ -13,15 +13,16 @@ import { readableReport } from "@pagopa/ts-commons/lib/reporters";
 
 export type IConfig = t.TypeOf<typeof IConfig>;
 export const IConfig = t.interface({
-  WALLET_CONFIG_API_HOST: NonEmptyString,
-  WALLET_CONFIG_WEBVIEW_PM_HOST: NonEmptyString,
   WALLET_CONFIG_API_BASEPATH: NonEmptyString,
-  WALLET_CONFIG_API_PM_BASEPATH: NonEmptyString,
-  WALLET_OUTCOME_API_BASEPATH: NonEmptyString,
   WALLET_CONFIG_API_ENV: NonEmptyString,
+  WALLET_CONFIG_API_HOST: NonEmptyString,
+  WALLET_CONFIG_API_PM_BASEPATH: NonEmptyString,
   WALLET_CONFIG_API_TIMEOUT: t.number,
+  WALLET_CONFIG_WEBVIEW_PM_HOST: NonEmptyString,
   WALLET_GDI_CHECK_TIMEOUT: t.number,
-  WALLET_NPG_SDK_URL: NonEmptyString
+  WALLET_NPG_SDK_URL: NonEmptyString,
+  WALLET_OUTCOME_API_BASEPATH: NonEmptyString,
+  WALLET_PAGOPA_LOGOS_CDN: NonEmptyString
```

Comment:
I think it's better to add the new env WALLET_PAGOPA_LOGOS_CDN variable also to the file README

Reply:
I've updated the Readme with the new env variable